tests/providers/amazon/aws/operators/test_s3_bucket.py

```diff
@@ -0,0 +1,84 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+import unittest
+
+import mock
+from moto import mock_s3
+
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+from airflow.providers.amazon.aws.operators.s3_bucket import S3CreateBucketOperator, S3DeleteBucketOperator
+
+BUCKET_NAME = os.environ.get("BUCKET_NAME", "test-airflow-bucket")
+TASK_ID = os.environ.get("TASK_ID", "test-s3-operator")
+
+
+class TestS3CreateBucketOperator(unittest.TestCase):
+    def setUp(self):
+        self.create_bucket_operator = S3CreateBucketOperator(
+            task_id=TASK_ID,
+            bucket_name=BUCKET_NAME,
+        )
+
+    @mock_s3
```

Comment:
Theoretically you don't need to `@mock_s3` anymore because you are mocking the Hook calls. So the calls to boto will not come through anyway. But that's fine.

Reply:
No so if we do not use this boto will throw credentials not found. The other way to solve this is by mocking connection as well which I think is a over kill when a wrapper is sufficient ;)